editor/editor_file_system.cpp

```diff
@@ -1979,6 +1973,20 @@ Error EditorFileSystem::_resource_import(const String &p_path) {
 	return OK;
 }
 
+bool EditorFileSystem::_should_skip_directory(const String &p_path) {
+	if (FileAccess::exists(p_path.plus_file("project.godot"))) {
```

Comment:
what is the scenario where there is a project within a project? Sounds dangerous

Reply:
It's specifically to avoid exploding when people do this kind of dangerous stuff, it's not a supported use case :)